Origam.PrintServer/GhostscriptPrinter.cs

```diff
@@ -0,0 +1,94 @@
+#region license
+/*
+Copyright 2005 - 2025 Advantage Solutions, s. r. o.
+
+This file is part of ORIGAM (http://www.origam.org).
+
+ORIGAM is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+ORIGAM is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with ORIGAM. If not, see <http://www.gnu.org/licenses/>.
+*/
+#endregion
+
+using System.Diagnostics;
+using System.Text;
+
+namespace Origam.PrintServer;
+
+public class GhostscriptPrinter(
+    IConfiguration config, 
+    ILogger<GhostscriptPrinter> log)
+{
+	private static readonly int ProcessTimeout = 60_000; //ms
```

Comment:
this looks like a TAB character at the begining of the line instead of white spaces